packages/patternfly-4/react-core/src/components/Select/CheckboxSelectOption.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Select/select';
+import { default as checkStyles } from '@patternfly/react-styles/css/components/Check/check';
+import { css } from '@patternfly/react-styles';
+import { SelectConsumer, KeyTypes } from './selectConstants';
+
+export interface CheckboxSelectOptionProps {
```

Comment:
missing `extends React.HTMLProps<HTMLElement>`.

Reply:
This errors out, but extending `React.HTMLProps<HTMLLabelElement>` works.

Reply:
okay that makes sense. I wasn't sure if the consumer would effect which type of element it expects.